game-core/src/test/java/games/strategy/engine/chat/ChatPlayerPanelTest.java

```diff
@@ -0,0 +1,31 @@
+package games.strategy.engine.chat;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.*;
+
+import java.awt.Dimension;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+public class ChatPlayerPanelTest {
+
+    @Mock private PlayerChatRenderer renderer;
+
+    private ChatPlayerPanel chatPlayerPanel;
+
+    @Test
+    public void setPlayerRenderer_whenChatIsNull_thenDynamicPreferredSizeIsSet() {
```

Comment:
Codacy found an issue: [The JUnit 5 test method name 'setPlayerRenderer_whenChatIsNull_thenDynamicPreferredSizeIsSet' doesn't match '[a-z][a-zA-Z0-9]*'](https://app.codacy.com/gh/triplea-game/triplea/pullRequest?prid=5821454)